src/lib/index.js

```diff
@@ -1145,6 +1176,14 @@ function templateFormatString(string, labels, d3locale) {
             }
         }
 
+        // Apply mult/div operation (if applicable)
+        if((parsedOp === '*' || parsedOp === '/') && value !== undefined) {
+            value = {
+                '*': (function(v) { return v * parsedNumber; }),
+                '/': (function(v) { return v / parsedNumber; }),
+            }[parsedOp](value);
+        }
```

Comment:
Wondering if we could simplify this logic without those functions? Something like: 
```js
        if(value !== undefined)  {
            if(parsedOp === '*') value *= parsedNumber;
            if(parsedOp === '/') value /= parsedNumber;
        }
```

Reply:
Done!